src/accessories/TokenArranger.ttslua

```diff
@@ -131,81 +142,73 @@ end
-  buttonParameters.label          = ""
-  buttonParameters.tooltip        = "Increase / Decrease"
-  buttonParameters.color          = { 0, 0, 0, 0 }
-  buttonParameters.width          = 325
-  buttonParameters.height         = 325
-  -- create UI
+-- creates buttons and inputs (if argument is true)
+function createButtonsAndInputs(loadInputs)
   local offset = 0.725
-  local pos = { x = { -1.067, 0.377 }, z = -2.175 }
+  local pos    = { x = { -1.067, 0.377 }, z = -2.175 }
 
   -- button and inputs index 0-9
   for i = 1, 10 do
     if i < 6 then
       buttonParameters.position = { pos.x[1], 0, pos.z + i * offset }
-      if loadInputs then
-        inputParameters.position  = { pos.x[1] + offset, 0.1, pos.z + i * offset }
-      end  
+      inputParameters.position  = { pos.x[1] + offset, 0.1, pos.z + i * offset }
     else
       buttonParameters.position = { pos.x[2], 0, pos.z + (i - 5) * offset }
-      if loadInputs then
-        inputParameters.position  = { pos.x[2] + offset, 0.1, pos.z + (i - 5) * offset }
-      end     
+      inputParameters.position  = { pos.x[2] + offset, 0.1, pos.z + (i - 5) * offset }  
     end
 
-    buttonParameters.click_function = attachIndex("tokenClick", i)
+    buttonParameters.click_function  = attachIndex("tokenClick", i)
     self.createButton(buttonParameters)
 
+    -- only create inputs on initial load
     if loadInputs then
-      inputParameters.input_function  = attachIndex2("tokenInput", i)
-      inputParameters.value           = tokenPrecedence[TOKEN_NAMES[i]][1]
+      inputParameters.input_function = attachIndex2("tokenInput", i)
+      inputParameters.value          = tokenPrecedence[TOKEN_NAMES[i]][1]
       self.createInput(inputParameters)
     end
-    
   end
 
   -- index 10: "Update / Hide" button
-  buttonParameters.label          = "Update / Hide"
-  buttonParameters.click_function = "layout"
-  buttonParameters.tooltip        = "Left-Click: Update!\nRight-Click: Hide Tokens!"
-  buttonParameters.position       = { 0.725, 0.1, 2.025 }
-  buttonParameters.color          = { 1, 1, 1 }
-  buttonParameters.width          = 675
-  buttonParameters.height         = 175
-  self.createButton(buttonParameters)
+  self.createButton({
+    function_owner = self,
+    label          = "Update / Hide",
+    click_function = "layout",
+    tooltip        = "Left-Click: Update!\nRight-Click: Hide Tokens!",
+    position       = { 0.725, 0.1, 2.025 },
+    color          = { 1, 1, 1 },
+    width          = 675,
+    height         = 175
+  })
 end
 
 -- update input fields
 function updateUI()
   for i = 1, 10 do
-    self.editInput({ index = i - 1, value = tokenPrecedence[TOKEN_NAMES[i]][1] })
+    self.editInput({
+      index = i - 1,
+      value = tokenPrecedence[TOKEN_NAMES[i]][1]
+    })
   end
 end
 
 -- order function for data sorting
-function token_value_comparator(left, right)
+function tokenValueComparator(left, right)
   if left.value > right.value then
```

Comment:
Two separate readability suggestions here:
Keep the left/right consistent all the way down:
if left.value > right.value then...
elseif left.value < right.value

The gt/lt and order flip is easy to miss

Whole thing could be a little more compact (and avoid the above issue) as:
if (left.value ~= right.value) then
  return left.value > right.value
elseif left.order ~= right.order
  return left.order < right.order
else
  return left.token.getGUID() > right.token.getGUID()
end
Though I don't think the GUID comparison is necessary, since identical tokens shouldn't really matter what order they come out in, and if the left/right flips it shouldn't actually change anything?  But if there's something attached to them where a shift could actually impact something it's fine to leave